src/Data/Bin/Properties.agda

```diff
@@ -121,14 +121,14 @@ suc≢0 {2[1+ _ ]} ()
 suc≢0 {1+[2 _ ]} ()
 
 pred-suc : pred ∘ suc ≗ id
-pred-suc zero      = refl
-pred-suc 2[1+ x ]  = sym (2[1+-as∘ x)
-pred-suc 1+[2 x ]  = refl
+pred-suc zero     =  refl
+pred-suc 2[1+ x ] =  sym (2[1+-as∘ x)
+pred-suc 1+[2 x ] =  refl
 
-suc-pred : ∀ {x} → x ≢ zero → suc (pred x) ≡ x
-suc-pred {zero}     0≢0  = contradiction refl 0≢0
-suc-pred {2[1+ _ ]} _    = refl
-suc-pred {1+[2 x ]} _    = sym (1+[2-as∘ x)
+suc-pred : ∀ x → x ≢ zero → suc (pred x) ≡ x
```

Comment:
Why have you made this argument explicit?

Reply:
By missing the point. Changed to implicit.